tests/Tests/600__ai_apps/680__starburst/681__managed_starburst_monitoring.robot

```diff
@@ -0,0 +1,58 @@
+*** Settings ***
+Documentation       Test suite testing SERH Metrics
+Resource            ../../../Resources/RHOSi.resource
+Resource            ../../../Resources/ODS.robot
+Resource            ../../../Resources/Common.robot
+
```

Comment:
could you add the RHOSi Setup/Teardown as Suite Setup and Teardown?

Reply:
so we need it for this TC ?